rpc_jobs/multi_node_aio.yml

```diff
@@ -1,37 +1,145 @@
       - mitaka:
           branch: mitaka-13.1
           USER_VARS: |
             tempest_test_sets: 'scenario defcore api heat_api smoke'
+          DEPLOY_TELEGRAF: "yes"
       - newton140:
           branch: newton-14.0
           USER_VARS: |
             tempest_test_sets: 'all'
+          DEPLOY_TELEGRAF: "yes"
       - newton141:
           branch: newton-14.1
           USER_VARS: |
             tempest_test_sets: 'all'
+            lxc_container_vg_name: "USE_DIR_NOT_VG"
+            glance_default_store: "swift"
+            neutron_legacy_ha_tool_enabled: true
+          UPGRADE_FROM_REF: "kilo"
+          DEPLOY_TELEGRAF: "yes"
       - master:
           branch: master
           USER_VARS: |
             tempest_test_sets: 'all'
-    context:
+          DEPLOY_TELEGRAF: "yes"
+    image:
       - xenial:
           DEFAULT_IMAGE: "16.04.2"
       - trusty:
           DEFAULT_IMAGE: "14.04.5"
+    action:
+      - deploy:
+          ACTION_STAGES: >-
+            Prepare MaaS,
+            Setup MaaS,
+            Verify MaaS,
+            Install Tempest,
+            Tempest Tests,
+            Prepare Kibana Selenium,
+            Kibana Tests,
+            Holland
+      - leapfrogupgrade:
+          ACTION_STAGES: >-
+            Prepare MaaS,
+            Setup MaaS,
+            Install Tempest,
+            Leapfrog Upgrade
+    resources:
+      - none:
+          GENERATE_TEST_NETWORKS: "0"
+          GENERATE_TEST_SERVERS: "0"
+          GENERATE_TEST_VOLUMES: "0"
+          COMPUTE_NODES: "2"
+          VOLUME_NODES: "2"
+      - small:
+          GENERATE_TEST_NETWORKS: "20"
+          GENERATE_TEST_SERVERS: "20"
+          GENERATE_TEST_VOLUMES: "8"
+          COMPUTE_NODES: "2"
+          VOLUME_NODES: "2"
+      - medium:
+          GENERATE_TEST_NETWORKS: "50"
+          GENERATE_TEST_SERVERS: "50"
+          GENERATE_TEST_VOLUMES: "20"
+          COMPUTE_NODES: "5"
+          VOLUME_NODES: "2"
+      - large:
+          GENERATE_TEST_NETWORKS: "100"
+          GENERATE_TEST_SERVERS: "100"
+          GENERATE_TEST_VOLUMES: "40"
+          COMPUTE_NODES: "10"
+          VOLUME_NODES: "2"
     trigger:
       - periodic:
           CRON: "H H(0-8) * * 1-5"
       - post-merge:
           CRON: ""
     exclude:
+      # Xenial builds are run for newton and above
+      # as it is not supported distro before newton.
+      - series: kilo
+        image: xenial
+      - series: liberty
+        image: xenial
       - series: mitaka
-        context: xenial
+        image: xenial
+      # Leapfrog upgrades are only run for kilo --> newton141
+      # as the upgrade method is not supported for any
+      # other target series.
+      # the kilo --> newton141 job is triggered by PRs to either branch.
+      - series: liberty
+        action: leapfrogupgrade
+      - series: mitaka
+        action: leapfrogupgrade
+      - series: newton140
+        action: leapfrogupgrade
+      - series: master
```

Comment:
I've copied this from rpc_aio.yml. While master might only be xenial now, I'd rather keep to to files consistent. It also doesn't hurt that it's explicit.